### PR TITLE
feat: kubectl plugin support info get the secret

### PR DESCRIPTION
**Pull request by jiuker**
support #1686 
Do the same thing like `kubectl get tenant tenantName -n namesspace -o=jsonpath="{.spec.configuration.name}" | xargs -I {} kubectl get secret {} -n
namesspace -o=jsonpath="{.data.config\\.env}" | base64 -d`

### CODE REVIEW

**Comment by shtripat** [comment]
I feel this could be part of `kubectl-minio tenant info` command itself. Also format of output could be as below

```
MinIO Root User CRedentials:
MINIO_ROOT_USER="JRAJG2B1H7EB35803XH8"
MINIO_ROOT_PASSWORD="nSrF2KRjV3ubqlkkdztSA0b4E2yN7mLux9IWFDM9"
```

**Comment by jiuker** [comment]
@harshavardhana cc

File: kubectl-minio/cmd/tenant-info.go [hunk 1]
```diff
@@ -93,10 +100,42 @@ func (d *infoCmd) run(args []string) error {
...
+	}
+	printTenantCredentials(string(secret.Data["config.env"]))
 	return nil
 }
 
+func printTenantCredentials(data string) {
+	users := regexp.MustCompile(`MINIO_ROOT_USER="([^"])+"`).FindAllString(data, -1)
+	passs := regexp.MustCompile(`MINIO_ROOT_PASSWORD="([^"])+"`).FindAllString(data, -1)
+	if len(users) >= 1 || len(passs) >= 1 {
+		fmt.Println("MinIO Root User CRedentials:")
```
Comment (shtripat) [inline kubectl-minio/cmd/tenant-info.go:129]: Minor spelling `%s/CRedentials/Credentials/g`

**Review by shtripat — COMMENTED** [review]
Mostly looks good. A minor spelling correction.

File: kubectl-minio/cmd/tenant-info.go [hunk 2]
```diff
@@ -93,10 +100,42 @@ func (d *infoCmd) run(args []string) error {
 		return err
 	}
 	printTenantInfo(*tenant)
-
+	// show the secret
+	fmt.Println("\n")
```
Comment (shtripat) [inline kubectl-minio/cmd/tenant-info.go:104]: Just `fmt.Println()` is enough or add at beginning of line#129

**Review by shtripat — APPROVED** [review]
LGTM. Verified
```
./kubectl-minio tenant info tenant-1
Tenant 'tenant-1', Namespace 'tenant-ns', Total capacity 16 GiB

Current status: Initialized
MinIO version: minio/minio:RELEASE.2023-06-23T20-26-00Z
MinIO service: minio/ClusterIP (port 443)
Console service: tenant-1-console/ClusterIP (port 9443)

POOL	SERVERS	VOLUMES(SERVER)	CAPACITY(VOLUME) 
0   	4      	1              	4.0 GiB         	

MinIO Root User Credentials:
MINIO_ROOT_USER="JRAJG2B1H7EB35803XH8"
MINIO_ROOT_PASSWORD="nSrF2KRjV3ubqlkkdztSA0b4E2yN7mLux9IWFDM9"
```

**Pull request merged**